nixos/extractors/services.nix

```diff
@@ -23,6 +23,9 @@
     optionalString
     optionalAttrs
     replaceStrings
+    forEach
+    removePrefix
+    splitString
```

Comment:
It might be a nitpick, but might be worth keeping these lexicographically sorted?